CONTRIBUTING.md

```diff
@@ -33,16 +39,28 @@ $ git clone git@github.com:<user>/kubebuilder.git $GOPATH/src/sigs.k8s.io/kubebu
 1. Build the project by using the command `make build` 
 1. Run the command `make install` to create a bin with the source code 
```

Comment:
`make install` already calls `make build` so the first is completely redundant (it will actually compile it twice which is a waste of time).

Reply:
It is true.